CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ Types of changes are:
 
 ## [Unreleased]
 
+- Pass through undefined options/arguments to mypy
```

Comment:
Please add header to the change to show what type of change it is (Added, Changed, etc...) This affects the version number when releasing.

Reply:
[b1937d8](https://github.com/radeklat/delfino/pull/35/commits/b1937d8f73f4cccc2325785b8611ba7de6ef8f43)